pkg/remote/aws/init.go

```diff
@@ -69,7 +69,7 @@ func Init(version string, alerter *alerter.Alerter,
 	supplierLibrary.AddSupplier(NewLambdaFunctionSupplier(provider, deserializer, lambdaRepository))
 	supplierLibrary.AddSupplier(NewDBSubnetGroupSupplier(provider, deserializer, rdsRepository))
 	supplierLibrary.AddSupplier(NewDBInstanceSupplier(provider, deserializer, rdsRepository))
-	supplierLibrary.AddSupplier(NewVPCSecurityGroupSupplier(provider, deserializer))
+	supplierLibrary.AddSupplier(NewVPCSecurityGroupSupplier(provider, deserializer, ec2repository))
 	supplierLibrary.AddSupplier(NewIamUserSupplier(provider, deserializer))
```

Comment:
Just like what you did for `ec2repo`, can you add `iamrepo` as an argument to each IAM supplier here.